tests/test_login.py

```diff
@@ -48,69 +49,143 @@ class ActiniaUserTestCase(ActiniaResourceTestCaseBase):
-    #     self.assertEqual(rv.mimetype, "application/json", "Wrong mimetype %s"%rv.mimetype)
-    #
-    #     # Get the new user entry
-    #     rv = self.server.get(URL_PREFIX + '/users/thomas', headers=self.admin_auth_header)
-    #     #print("GET /users/thomas", rv.data)
-    #     self.assertEqual(rv.status_code, 200, "HTML status code is wrong %i"%rv.status_code)
-    #     self.assertEqual(rv.mimetype, "application/json", "Wrong mimetype %s"%rv.mimetype)
-    #
-    #     self.assertEquals(json_load(rv.data)["User id"], new_user_id)
-    #     self.assertEquals(json_load(rv.data)["User role"], "user")
-    #
-    #     # Get the admin user entry
-    #     rv = self.server.get(URL_PREFIX + '/users/admin', headers=self.admin_auth_header)
-    #     #print("GET /users/admin", rv.data)
-    #     self.assertEqual(rv.status_code, 200, "HTML status code is wrong %i"%rv.status_code)
-    #     self.assertEqual(rv.mimetype, "application/json", "Wrong mimetype %s"%rv.mimetype)
-    #
-    #     self.assertEquals(json_load(rv.data)["User id"], "admin")
-    #     self.assertEquals(json_load(rv.data)["User role"], "admin")
-    #
-    #     # Delete the user as admin
-    #     rv = self.server.delete(URL_PREFIX + '/users/%s' % new_user_id, headers=self.admin_auth_header)
-    #     print("DELETE /users/thomas", rv.data)
-    #     self.assertEqual(rv.status_code, 200, "HTML status code is wrong %i"%rv.status_code)
+    @pytest.mark.dev
+    def test_create_get_delete_user(self):
+        """
+        Create a new user, access it as admin and as normal user and delete it
+        :return:
+        """
+
+        # Create a new user
+        new_user_id = "thomas"
+        new_group = "test"
+        new_password = "12345"
+        self.created_users.append(new_user_id)
+
+        create_user_url = f"{URL_PREFIX}/users/{new_user_id}?" \
+            f"password={new_password}&group={new_group}"
+        rv = self.server.post(create_user_url, headers=self.admin_auth_header)
+        # print("POST /users/thomas", rv.data)
+        self.assertEqual(
+            rv.status_code,
+            201,
+            "HTML status code is wrong %i" % rv.status_code
+        )
+        self.assertEqual(
+            rv.mimetype,
+            "application/json",
+            "Wrong mimetype %s" % rv.mimetype
+        )
+
+        # Try to create the user again which should fail
+        rv = self.server.post(create_user_url, headers=self.admin_auth_header)
+        # print("POST /users/thomas", rv.data)
+        self.assertEqual(
+            rv.status_code,
+            400,
+            f"HTML status code is wrong {rv.status_code}"
+        )
+        self.assertEqual(
+            rv.mimetype,
+            "application/json",
+            "Wrong mimetype %s" % rv.mimetype
+        )
+
+        # Get the new user entry
+        rv = self.server.get(
+            URL_PREFIX + '/users/thomas', headers=self.admin_auth_header
```

Comment:
```suggestion
            URL_PREFIX + f"/users/{new_user_id}", headers=self.admin_auth_header
```
thomas might change his name to tina